roles/etcd_client_certificates/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Check status of external etcd certificatees
+  stat:
+    path: "{{ etcd_cert_config_dir }}/{{ item }}"
+  with_items:
+  - "{{ etcd_cert_prefix }}client.crt"
+  - "{{ etcd_cert_prefix }}client.key"
+  - "{{ etcd_cert_prefix }}ca.crt"
+  register: g_external_etcd_cert_stat_result
+  when: groups.oo_etcd_to_config is defined and groups.oo_etcd_to_config
+
+- set_fact:
+    etcd_client_certs_missing: "{{ False in (g_external_etcd_cert_stat_result.results
+                                   | oo_collect(attribute='stat.exists')
+                                   | list) }}"
+  when: groups.oo_etcd_to_config is defined and groups.oo_etcd_to_config
+
+- include: create.yml
+  when: groups.oo_etcd_to_config is defined and groups.oo_etcd_to_config and etcd_client_certs_missing | bool
```

Comment:
We should be passing in a variable containing the list of hosts instead of relying on groups.oo_etcd_to_config.